### ISSUE TITLE
Aggregated Residual Transformations for Deep Neural Networks

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#17]
## Main Idea
**Cardinality, the best way to imporve performance in residual network**
Paper: <https://arxiv.org/abs/1611.05431>

## Abstract
We present a simple, highly modularized network architecture for image classification. Our network is constructed by repeating a building block that aggregates a set of transformations with the same topology. Our simple design results in a homogeneous, multi-branch architecture that has only a few hyper-parameters to set. This strategy exposes a new dimension, which we call “cardinality” (the size of the set of transformations), as an essential factor in addition to the dimensions of depth and width. On the ImageNet-1K dataset, we empirically show that even under the restricted condition of maintaining complexity, increasing cardinality is able to improve classification accuracy. Moreover, increasing cardinality is more effective than going deeper or wider when we increase the capacity. Our models, named ResNeXt, are the foundations of our entry to the ILSVRC 2016 classification task in which we secured 2nd place. We further investigate ResNeXt on an ImageNet-5K set and the COCO detection set, also showing better results than its ResNet counterpart. The code and models are publicly available online.

## Contribution
+ 기존 ResNet에서 사용하던 width, depth와는 다른 'cardinality' 라는 개념을 도입, ResNeXt 모델 제시
+ Block들은 모두 동일한 hyper-parameter를 가짐, FLOP의 관점에서 연산적 복잡성을 고려한 것으로 spatial map은 2배로 downsample될때마다 block의 width는 2배로 증가시킴 => 모든 block이 동일한 수준의 복잡도
+ Inception-ResNet block과 유사, 모든 path가 동일한 topology를 적용한다는 차이 => 각 path에 대한 design이 용이
+ 같은 복잡도를 같는 ResNets와 비교 => ResNeXt가 더 높은 성능
+ Width와 depth의 증가 대신 channel을 몇개의 그룹으로 분할하는 'cardinality'를 적용하는 것이 더 효과적
+ 2016년 ILSVRC classification task에서 2위를 달성

## Reviews
<https://deepkerry.tistory.com/29>
